lib/components/checkbox/checkbox.dart

```diff
@@ -1,23 +1,55 @@
 import 'package:flutter/material.dart';
 import 'package:mix/mix.dart';
-import 'package:remix_ui/components/checkbox/checkbox.style.dart';
+
 import 'package:remix_ui/components/checkbox/checkbox.variants.dart';
 
-import '../../utils/component_recipe.dart';
+import 'tokens/checkbox_spec.dart';
+import 'tokens/checkbox_util.dart';
+
+final $defaultCheckboxStyle = AnimatedStyle(
```

Comment:
Lets just declare a `Style`, and then when we use it, we can do `style.animate()`, i think this becomes a bit cleaner